gaffer-as-a-service/gaas-rest/src/main/java/uk/gov/gchq/gaffer/gaas/client/graph/ValidateGraphHostOperation.java

```diff
@@ -52,24 +52,24 @@ public void execute() throws GraphOperationException {
             validateSystemIsUp(response);
 
         }  catch (final WebClientRequestException e) {
-            LOGGER.error(String.format("Get Status request for %s failed. Reason: %s at %s ", proxySubGraph.getGraphId(), e.getMostSpecificCause().getMessage(), e.getUri()), e);
+            LOGGER.error("Get Status request for {} failed. Reason: {} at {}", proxySubGraph.getGraphId(), e.getMostSpecificCause().getMessage(), e.getUri(), e);
             throw new GraphOperationException("Get Status request for '" + proxySubGraph.getGraphId() + "' failed. Reason: " + e.getMostSpecificCause().getMessage() + " at " + e.getUri(), e);
 
         } catch (final WebClientResponseException e) {
-            LOGGER.error(String.format("Get Status request for %s returned: %s %s at %s ", proxySubGraph.getGraphId(), e.getRawStatusCode(), e.getStatusText(), e.getRequest().getURI()), e);
-             throw new GraphOperationException("Get Status request for '" + proxySubGraph.getGraphId() + "' returned: " + e.getRawStatusCode() + " " + e.getStatusText() + " at " + e.getRequest().getURI(), e);
+            LOGGER.error(String.format("Get Status request for {} returned: {} {} at {} ", proxySubGraph.getGraphId(), e.getRawStatusCode(), e.getStatusText(), e.getRequest().getURI()), e);
```

Comment:
The `String.format` is [not needed](https://www.tutorialspoint.com/slf4j/slf4j_parameterized_logging.htm), I just got confused before but it does work with exceptions at the end still. See https://github.com/gchq/gaffer-experimental/pull/303#discussion_r880140216

Reply:
@t92549  Thank you for the input. please check my new updates